tests/TestCase.php

```diff
@@ -32,26 +35,29 @@ public static function prepareSession()
     {
         if (null === self::$mink) {
             $session = new Session(self::getConfig()->createDriver());
-            self::$mink = new Mink(['sess' => $session]);
+            self::$mink = new Mink([self::MINK_SESSION_KEY => $session]);
         }
     }
 
     /**
-     * @return AbstractConfig
-     *
-     * @throws \UnexpectedValueException if the global driver_config_factory returns an invalid object
```

Comment:
the description should probably be kept

Reply:
I wasn't sure if it would make sense to move it to `createConfig` or have it in both cases, but now that I think about it, it doesn't make sense at all: this is a private method of a class (not a trait even) so we don't really need to document it. Plus, the exception message is understandable enough.

Reply:
I'm also for keeping the description. It's now always clear when an exception is thrown just by looking at the DocBlock itself.

If the method is private it doesn't mean it should be less documented, because same documentation is also used by maintainers.

Reply:
> I'm also for keeping the description. It's now always clear when an exception is thrown just by looking at the DocBlock itself.

That will only apply to `createConfig()` which I expect to be rarely seen. `getConfig` would be more common and protected/public usages of `getConfig` even more than that.

If we care about documentation, then we should mention that exception in all cases (like I said, especially the API ones) not just to the internal one that no will ever actually see.

Otherwise I don't see the point of a 4-line comment explaining verbatim what the next 10 lines of code clearly do.